code/game/objects/items/weapons/twohanded.dm

```diff
@@ -248,6 +248,59 @@
 	sharp = IS_SHARP_ITEM_SIMPLE
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "stabbed", "jabbed", "torn", "gored")
+	var/current_angle = 45	//Based on what direction the tip of the spear is pointed at in the sprite; maybe someone makes a spear that points northwest
+
+/obj/item/weapon/twohanded/spear/throw_at(atom/target, range, speed, thrower, spin, flying)
+	spin = FALSE
+	//Find the angle the spear is to be thrown at, then rotate it based on that angle at 45 degree increments
```

Comment:
This seems hacky but ok

Reply:
Lumi recommended finding the precise angle as it has no impact performance. Sprite can look bad at some odd angles so it's better to find the precise angle and then turn it in any of the 8 directions. Could probably add another 8 directions so that it turns at every 22.5 degrees but that may be as far as it goes before the player gets confused at what they are looking at (in my opinion).

Reply:
I meant just use the precise angle full stop. There's not really a point in getting the precise angle but then just changing it into a cardinal angle anyway.

Reply:
Problem is it looks bad when it is not a cardinal angle. Sprites are important for conveying info. Imagine a player looking for their spear in the heat of battle and missing it because it was turned at a 68 degree angle and didn't look like a spear anymore.